#7981 datadir doc improved (#8098)
geosolutions-it/MapStore2: docs/developer-guide/externalized-configuration.md

```json
[{"op": "add", "path": "/plugins/desktop/-", "value": "MyAwesomePlugin"}]
```

## Externalize front-end Configurations

> From version 2021.02.xx, the externalization of the front-end files is automatic on the back-end. As well as you configure the data-directory.

Reviewer: ```suggestion
From version 2021.02.xx, the externalization of the front-end files is automatic on the back-end, as well as you configure the data-directory.
```

> Anyway for your custom application you can customize the following paths to change the defaults and implement your own services for configuration, extensions, and so on.

Reviewer: ```suggestion
Anyway for your custom application you may want customize the following paths to change the defaults and implement your own services for configuration, extensions, and so on.
```

> Anyway for your custom application you can customize the following paths to change the defaults and implement your own services for configuration, extensions, and so on.
> 
> You can externalize the following files to the data directory by adding the relative line in the `app.jsx` :

Reviewer: ```suggestion
Anyway for your custom application you may want to customize the following paths to point your own services for configuration, extensions, and so on.
The paths can be customized by adding the relative line in the `app.jsx` :
```
